packages/shared/selectors/k8s.ts

```diff
@@ -49,3 +49,5 @@ export const getInfrastructurePlatform = (
 export const getAPIVersion = <A extends K8sResourceCommon = K8sResourceCommon>(
   value: A
 ) => _.get(value, 'apiVersion') as K8sResourceCommon['apiVersion'];
+
+export const getErrorMessage = (error: Error) => error?.message;
```

Comment:
```suggestion
export const getErrorMessage = (error: Error) => error?.message;
```
Not a generic K8s selector move it to a different file.